### PR TITLE
Add GetDefaultTarget and SetDefaultTarget APIs

### LINKED ISSUE #944: RFE: Add GetDefaultTarget and SetDefaultTarget API
## Please describe what you would like to see

In order to be able to get and set the default target of the system to boot into, it is necessary to extend BlueChi with the `GetDefaultTarget` and `SetDefaultTarget` API provided by the systemd Manager API (see the [freedesktop systemd documentation](https://www.freedesktop.org/wiki/Software/systemd/dbus/)). 
For more information about the boot process, see [here](https://www.freedesktop.org/software/systemd/man/latest/bootup.html#System%20Manager%20Bootup).

## Please describe the solution you'd like

- [x] `bluechi-controller`: Add new API methods on public DBus for interface `org.eclipse.bluechi.Node` and delegate to `bluechi-agent`
   - [x] `GetDefaultTarget`
   - [x] `SetDefaultTarget`
- [x] `bluechi-agent`: Add new API methods on internal DBus and delegate to systemd bus
   - [x] `GetDefaultTarget`
   - [x] `SetDefaultTarget`
- [x] `bluechictl`: Add new commands in bluechictl:
   - [x] `bluechictl get-default [nodename]` 
   Similar to [systemctl get-default](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#get-default)). `[nodename]` is required for now. Maybe later this can be changed - if `[nodename]` is not provided, print all default targets for all nodes
   - [x] `bluechictl set-default [nodename] [target]`
   Similar to [systemctl set-default TARGET](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#set-default%20TARGET). 
   The `target` needs to be a `.target` unit, so there should be pre-validation in place checking that the unit ends with `.target`.
- [x] Implement integration tests
- [x] Extend documentation (man pages, examples, etc.)

### PR DESCRIPTION
**Pull request by ArtiomDivak**
- Adds support for GetDefaultTarget and  SetDefaultTarget APIs on controller and node
- Adds get-default and set-default commands to bluechictl
- Adds integration tests for both APIs

Fixes: https://github.com/eclipse-bluechi/bluechi/issues/944
Signed-off-by: Artiom Divak <adivak@redhat.com>

### CODE REVIEW

File: tests/bluechi_test/bluechictl.py [hunk 1]
```diff
@@ -149,6 +149,36 @@ def reset_failed(
             expected_result,
         )
 
+    def get_default_target(
+        self,
+        node_name: str = "",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"get-default {node_name}".strip()
```
Comment (mwperina) [inline tests/bluechi_test/bluechictl.py]: Do we really need to execute `strip()` here?

File: tests/bluechi_test/bluechictl.py [hunk 2]
```diff
@@ -149,6 +149,36 @@ def reset_failed(
...
+
+    def set_default_target(
+        self,
+        node_name: str = "",
+        target: str = "",
+        force: str = "false",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"set-default {node_name} {target} {force}".strip()
```
Comment (mwperina) [inline tests/bluechi_test/bluechictl.py]: Same as above, is `strip()` really needed?

File: tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
+
+    node_foo.exec_run("systemctl set-default multi-user.target")
```
Comment (mwperina) [inline tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py]: Do we need to run `set-default` as a part of the test? Isn't it enough for testing get-default just to compare `bluechictl` and `systemctl` results?

File: tests/tests/tier0/bluechi-get-default-target/main.fmf
Comment (mwperina) [inline tests/tests/tier0/bluechi-get-default-target/main.fmf:1]: I'd rename the directory to `bluechictl-get-default-target`

File: tests/tests/tier0/bluechi-set-default-target/main.fmf
Comment (mwperina) [inline tests/tests/tier0/bluechi-set-default-target/main.fmf:1]: I'd rename the directory to `bluechictl-set-default-target`

File: src/client/method-get-default-target.c
Comment (mwperina) [inline src/client/method-get-default-target.c:1]: get and set default target functionality is very closely related, wouldn't it make more sense to have both function in a single file (for example `method-default-target.{c|h}`?

**Review by mwperina — COMMENTED** [review]
Generally it looks great, some small nits posted

**Comment by mwperina** [comment]
Also please don't forget to update bluechictl man page: https://github.com/eclipse-bluechi/bluechi/blob/main/doc/man/bluechictl.1.md

File: tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py (hunk 3 above)
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py]: I am not sure about this one. I feel like I need to see that if the default target changes the get default command will get the correct answer. But if you think it's unnecessary I will delete it.

Comment (mwperina) [reply tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py]: Well, for me checking get is enough.
But on the other hand we can even have a single test for both get and set:

1. Run both bluechictl and systemctl get and compare values
2. Run bluechictl set
3. Run both bluechictl and systemctl get and check if the new target is really reported by both

Having a test like this would verify both functions

Comment (ArtiomDivak) [reply tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py]: I thought about this test. But if this fails I am not sure we can know if it is a get or a set problem. Or maybe I am wrong?

File: tests/bluechi_test/bluechictl.py (hunk 1 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/bluechictl.py]: Changed

File: tests/bluechi_test/bluechictl.py (hunk 2 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/bluechictl.py]: Changed

File: tests/tests/tier0/bluechi-get-default-target/main.fmf
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechi-get-default-target/main.fmf:1]: Changed

File: tests/tests/tier0/bluechi-set-default-target/main.fmf
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechi-set-default-target/main.fmf:1]: Changed

File: src/client/method-get-default-target.c
Comment (ArtiomDivak) [reply src/client/method-get-default-target.c:1]: Changed

File: tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py (hunk 3 above)
Comment (mwperina) [reply tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py]: Step 1. verifies get functionality (bluechictl and systemctl should return the same value). 

Steps 2. and 3. verifiy set functionality. Step 2 operation should finish with success and then in step 3 we verifed that requested target was really applied and reported the same by both bluchictl and systemctl.

If something fails, then of course we would need to go to logs to find out which operation failed. But again we can have either a single test for both or specialized tests for each operation, I don't have a strong preference here

File: src/client/method-default-target.c [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        _cleanup_sd_bus_error_ sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_sd_bus_message_ sd_bus_message *message = NULL;
+        int r = 0;
+        bool force_b = 0;
+        if (!strcmp(force, "true")) {
+                force_b = 1;
+        } else if (!strcmp(force, "false")) {
+                force_b = 0;
+        } else {
+                fprintf(stderr, "Please provide a true/false value for force parameter\n");
```
Comment (mwperina) [inline src/client/method-default-target.c]: Neither man page nor `systemctl --help` mentions the force parameter for `set-default` command. Looking at systemctl source code https://github.com/systemd/systemd/blob/main/src/systemctl/systemctl-set-default.c#L130 they set force to true by default.

So shouldn't we set it to true as default (if not specified)?

File: src/client/method-help.c [hunk 5]
```diff
@@ -49,6 +49,10 @@ void usage_bluechi() {
         printf("    usage: daemon-reload nodename\n");
         printf("  - set-loglevel: change the log level of the bluechi-controller or a connected node\n");
         printf("    usage: set-loglevel [nodename] [loglevel]\n");
+        printf("  - get-default: get a default target of connected node\n");
+        printf("    usage: get-default [nodename]\n");
+        printf("  - set-default: set a default target of connected node\n");
+        printf("    usage: set-default [nodename] [target] [force]\n");
```
Comment (mwperina) [inline src/client/method-help.c]: As systemctl doesn't mention force parameter neither in manpage nor help, I'd remove the parameter from the help in bluechictl

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    multi_user_string = "multi-user.target"
+    graphical_string = "graphical.target"
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
```
Comment (mwperina) [inline tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py]: Why using `in` ? Shouldn't the results be equal?

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py
Comment (mwperina) [inline tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py:1]: Please fix typo in file name

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        ctrl.bluechictl.set_default_target(NODE_FOO, multi_user_string, "false")
+        _, systemd_result = node_foo.exec_run("systemctl get-default")
+        assert systemd_result == multi_user_string
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
+
+
+def test_bluechi_set_default_target(
```
Comment (mwperina) [inline tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py]: Please align the method name with file name

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py (hunk 6 above)
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py]: The result that I get in bc_result is `The default target of this node is: graphical.target`.
That is why I used `in`

File: src/client/method-default-target.c (hunk 4 above)
Comment (ArtiomDivak) [reply src/client/method-default-target.c]: Changed

File: src/client/method-help.c (hunk 5 above)
Comment (ArtiomDivak) [reply src/client/method-help.c]: Changed

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py:1]: Changed

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py (hunk 7 above)
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py]: Changed

File: src/client/method-default-target.c [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+                return r;
+        }
+
+        r = sd_bus_message_read(message, "s", &result);
+        if (r < 0) {
+                fprintf(stderr, "Failed to parse response message: %s\n", strerror(-r));
+                return r;
+        }
+
+        printf("The default target of this node is: %s\n", result);
```
Comment (mwperina) [inline src/client/method-default-target.c]: You should print here only the returned target to align output with systemctl

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py (hunk 6 above)
Comment (mwperina) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py]: See my other comment, that's your message, but to align with systemctl you should print only the returned target

Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py]: OK

File: src/client/method-default-target.c [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+        return r;
+}
+
+
+static int method_set_default_target_on(Client *client, char *node_name, char *target, char *force) {
+        _cleanup_sd_bus_error_ sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_sd_bus_message_ sd_bus_message *message = NULL;
+        int r = 0;
+        bool force_b = 1;
+        if (!strcmp(force, "false")) {
```
Comment (mwperina) [inline src/client/method-default-target.c]: Hmm, shouldn't the condition be reversed?

```
bool force_b = 1;
if (streq(force, "false")) {
         force_b = 0;
}
```

File: doc/man/bluechictl.1.md [hunk 10]
```diff
@@ -136,3 +136,11 @@ Fetches the status of the systemd units for the `bluechi-agent`.
 ### **bluechictl** *reset-failed* [*agent*] [*unit1*,*...*]
 
 Performs a `reset-failed` on the chosen `bluechi-agent` for the selected units.
+
+### **bluechictl** *get-default* [*agent*] 
+
+Fetches the default target on the chosen `bluechi-agent`.
+
+### **bluechictl** *set-default* [*agent*] [file.target]
```
Comment (mwperina) [inline doc/man/bluechictl.1.md]: Wouldn't `TARGET` be more readable than `file.target`?

File: src/client/method-default-target.c [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+void usage_method_get_default_target() {
+        usage_print_header();
+        usage_print_usage("bluechictl get-default [nodename]");
+}
+
+void usage_method_set_default_target() {
+        usage_print_header();
+        usage_print_usage("bluechictl set-default [nodename]");
```
Comment (mwperina) [inline src/client/method-default-target.c]: Shouldn't there be target parameter mentioned?

```
        usage_print_usage("bluechictl set-default [nodename] [target]");
```

File: tests/bluechi_test/bluechictl.py [hunk 12]
```diff
@@ -168,6 +168,36 @@ def reset_failed(
...
+
+    def set_default_target(
+        self,
+        node_name: str = "",
+        target: str = "",
+        force: str = "true",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"set-default {node_name} {target} {force}"
```
Comment (mwperina) [inline tests/bluechi_test/bluechictl.py]: Force parameter is no longer supported by bluechictl, so please remove it

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    multi_user_string = "multi-user.target"
+    graphical_string = "graphical.target"
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
```
Comment (mwperina) [inline tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py]: Please add this command to https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/systemctl.py and re-use it here

File: tests/tests/tier0/bluechictl-set-get-default-target/main.fmf [hunk 14]
```diff
@@ -0,0 +1,2 @@
+summary: Test the set and get default target function
```
Comment (mwperina) [inline tests/tests/tier0/bluechictl-set-get-default-target/main.fmf]: I'd rephrase to `Test bluechictl get-default and set-default commands`

**Review by mwperina — CHANGES_REQUESTED** [review]

File: src/client/method-default-target.c (hunk 9 above)
Comment (ArtiomDivak) [reply src/client/method-default-target.c]: Changed

File: doc/man/bluechictl.1.md (hunk 10 above)
Comment (ArtiomDivak) [reply doc/man/bluechictl.1.md]: Changed

File: src/client/method-default-target.c (hunk 11 above)
Comment (ArtiomDivak) [reply src/client/method-default-target.c]: Changed

File: tests/bluechi_test/bluechictl.py (hunk 12 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/bluechictl.py]: Changed

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py (hunk 13 above)
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py]: Changed

File: tests/tests/tier0/bluechictl-set-get-default-target/main.fmf (hunk 14 above)
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/main.fmf]: Changed

File: src/client/main.c [hunk 15]
```diff
@@ -42,27 +43,28 @@ int method_version(UNUSED Command *command, UNUSED void *userdata) {
 }
 
 const Method methods[] = {
-        { "help",            0, 0,       OPT_NONE,                                method_help,            usage_bluechi                },
-        { "list-unit-files", 0, 1,       OPT_FILTER,                              method_list_unit_files, usage_method_list_unit_files },
-        { "is-enabled",      2, 2,       OPT_NONE,                                method_is_enabled,      usage_method_is_enabled      },
```
Comment (mkemel) [inline src/client/main.c:47]: You have removed `is-enabled` method, this is why integration test is failing

File: tests/bluechi_test/bluechictl.py (hunk 12 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/bluechictl.py]: I just want to clarify this part of the code.
Should I only ask for two parameters (node, target) in the `set-default` command?

Comment (mwperina) [reply tests/bluechi_test/bluechictl.py]: I'd leave the parameter in API (same as systemd), but for bluechictl I wouldn't mention the force parameter and set it to true by default (same as systemctl does)

File: tests/bluechi_test/systemctl.py [hunk 16]
```diff
@@ -94,12 +94,20 @@ def restart_unit(
...
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self.client.exec_run(f"{SystemCtl.binary_name} get-default")
+
     def reset_failed_for_unit(
         self, unit_name: str, check_result: bool = True, expected_result: int = 0
     ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
-        return self._do_operation_on_unit(
-            unit_name, "reset-failed", check_result, expected_result
-        )
+        return self.client.exec_run(f"{SystemCtl.binary_name} daemon-reload")
```
Comment (mwperina) [inline tests/bluechi_test/systemctl.py]: What is this change for? Isn't it only an error in rebase?

File: tests/bluechi_test/systemctl.py [hunk 17]
```diff
@@ -94,12 +94,20 @@ def restart_unit(
             unit_name, "restart", check_result, expected_result
         )
 
+    def set_default_target(
+        self, target: str
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self.client.exec_run(f"{SystemCtl.binary_name} set-default {target}")
```
Comment (mwperina) [inline tests/bluechi_test/systemctl.py]: Please use `_do_operation` method

File: tests/bluechi_test/systemctl.py [hunk 18]
```diff
@@ -94,12 +94,20 @@ def restart_unit(
...
+
+    def get_default_target(
+        self,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self.client.exec_run(f"{SystemCtl.binary_name} get-default")
```
Comment (mwperina) [inline tests/bluechi_test/systemctl.py]: Please use `_do_operation` method

File: src/client/main.c (hunk 15 above)
Comment (ArtiomDivak) [reply src/client/main.c:47]: Changed

File: tests/bluechi_test/systemctl.py [hunk 19]
```diff
@@ -94,12 +94,20 @@ def restart_unit(
             unit_name, "restart", check_result, expected_result
         )
 
+    def set_default_target(
+        self, target: str
```
Comment (mwperina) [inline tests/bluechi_test/systemctl.py]: Does it make sense to check result code for this operation? If so, then please align with other methods in this module

File: tests/bluechi_test/systemctl.py [hunk 20]
```diff
@@ -94,12 +94,20 @@ def restart_unit(
...
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self.client.exec_run(f"{SystemCtl.binary_name} set-default {target}")
+
+    def get_default_target(
```
Comment (mwperina) [inline tests/bluechi_test/systemctl.py:103]: Does it make sense to check result code for this operation? If so, then please align with other methods in this module

File: src/client/main.c (hunk 15 above)
Comment (mwperina) [reply src/client/main.c:47]: No, method is mentioned below, it's just reformated

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    multi_user_string = "multi-user.target"
+    graphical_string = "graphical.target"
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.systemctl.get_default_target()
```
Comment (mwperina) [inline tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py]: Does it make sense also to check return codes?

**Review by mwperina — CHANGES_REQUESTED** [review]

File: src/client/main.c (hunk 15 above)
Comment (mkemel) [reply src/client/main.c:47]: @mwperina Artiom has re-added it, before that `is-enabled` wasn't there and the corresponding test failed because of that

Comment (mkemel) [reply src/client/main.c:47]: I think there was some rebase issue

Comment (mkemel) [reply src/client/main.c:47]: And it is still there! Looking at the commits, @ArtiomDivak you are adding `is-enabled` method, although it should be already there. Please do a proper rebase.

Comment (ArtiomDivak) [reply src/client/main.c:47]: I recently rebased and added the line. Now the issue is gone. Not sure what made it work

Comment (mkemel) [reply src/client/main.c:47]: Sorry, you have removed it in the first commit and added it in the second commit.

Comment (mkemel) [reply src/client/main.c:47]: Please just undo the removal in this commit https://github.com/eclipse-bluechi/bluechi/pull/951/commits/3e554d2db0dad610028823001df5212f49be38f7 rather than removing it and adding it in the next commit.

Or even better, just squash these two commits to one. You are just creating a commit that will fail if checked out. No need for this complication, work with a single commit per PR, it is good enough. If you decide to create separate commits, you need to constantly make sure that each commit is independent and the system will run correctly if built on top of each one of them.

Comment (mkemel) [reply src/client/main.c:47]: Great

File: tests/bluechi_test/systemctl.py (hunk 18 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/systemctl.py]: Changed

File: tests/bluechi_test/systemctl.py (hunk 17 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/systemctl.py]: Changed

File: tests/bluechi_test/systemctl.py (hunk 16 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/systemctl.py]: Changed

File: tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py (hunk 21 above)
Comment (ArtiomDivak) [reply tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_default_target.py]: Changed

File: tests/bluechi_test/systemctl.py (hunk 20 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/systemctl.py:103]: Changed

File: tests/bluechi_test/systemctl.py (hunk 19 above)
Comment (ArtiomDivak) [reply tests/bluechi_test/systemctl.py]: Changed

**Review by mwperina — APPROVED** [review]
LGTM

**Review by mkemel — APPROVED** [review]
LGTM
